src/containers/artist-dashboard-page/Tiers.tsx

```diff
@@ -50,6 +55,30 @@ export const audioTierMapPng: {
   platinum: <img alt='' src={IconPlatinumBadge} />
 }
 
+const BADGE_LOCAL_STORAGE_KEY = 'last_badge_tier'
+
+const useShowConfetti = (tier: BadgeTier) => {
+  // No tier or no local storage, never show confetti
+  if (tier === 'none' || !window.localStorage) return false
+
+  const lastBadge = window.localStorage.getItem(BADGE_LOCAL_STORAGE_KEY) as
+    | BadgeTier
+    | undefined
+
+  // set last tier
+  window.localStorage.setItem(BADGE_LOCAL_STORAGE_KEY, tier)
```

Comment:
might want to clear this on sign out. we probably should have some @audius notation and sweep all of them on sign out